index.html

```diff
@@ -0,0 +1,108 @@
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+  
+</head>
+<body>
+
+  <header>
+    <appbar class="appbar">
+        <ul class="appbar status">
+            <li>
+                <img src="sources/Time Style.svg"></img>
+            </li>
+            <li>
+                <img src="sources/Group 3.svg"></img>
+            </li>
+        </ul>
+        <nav class="appbar  navbar navbar-light fixed-top">
+          <div class="container-fluid">
+            <div class="nav-container">
+            <a class="navbar-brand" href="#">My Logo</a>
+            </div>
+            <button class="navbar-toggler" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvasNavbar" aria-controls="offcanvasNavbar">
+              <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="offcanvas offcanvas-end" tabindex="-1" id="offcanvasNavbar" aria-labelledby="offcanvasNavbarLabel">
+              
+              <div class="offcanvas-body">
+                <ul class="navbar-nav justify-content-end flex-grow-1 pe-3">
+                  <li class="nav-item">
+                    <a class="nav-link active" aria-current="page" href="#">Portfolio</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link " aria-current="page" href="#">About</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link " aria-current="page" href="#">Contact</a>
+                  </li>
+                                 
+                </ul>
+            
+              </div>
+            </div>
+          </div>
+        </nav>
+    </appbar>
+</header>
+
+  <div class="main">
+  <section class="mobile">
+      <h1 class="mobile heading">I’m Mariane
+      Glad to see <br> you!</h1>
+      <p id="paragraph1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <p id="paragraph2">LET’S CONNECT</p>
+
+      <div class="icons">
+         <a href="#"><img class="icon icon1" id="twitter"  src="sources/Vector4.png"></a>     
+         <a href="#"><img class="icon icon2" id="linkedin" src="sources/vector2.png"></a>     
+         <a href="#"><img class="icon icon3" id="medium"   src="sources/Vector5.png"></a>        
+         <a href="#"><img class="icon icon4" id="github"  src="sources/Vector1.png"></a>        
+         <a href="#"><img class="icon icon5" id="peace"  src="sources/Vector3.png"></a>   
+      
+         </div>
```

Comment:
- [ ] The list of icons should be grouped in a `ul` tag, and an `li` per icon; each icon should have a link (`a` tag). Kindly fix this 🙏. Example:
```
<ul>
  <li>
    <a>
      <img>
    </a>
  </li>
</ul>
```

- [ ] Please make sure all your `img` tags has an `alt` attribute this will make your site experience better and more accessible🙏